module/Activity/src/Activity/Controller/ActivityController.php

```diff
@@ -7,6 +7,7 @@
 use Zend\Mvc\Controller\AbstractActionController;
 use Zend\Session\Container as SessionContainer;
 use Activity\Form\ActivitySignup as SignupForm;
+use Activity\Form\ModifyRequest as RequestForm;
```

Comment:
It seems I had forgotten to add it. Retrieving form instances from the service is a good idea in general, but does not benefit this particular form (no instances living in the service are required) and this has not been applied yet in the rest of the module, so I think that that can be deferred.